wireapp/wire-android
# Rewrite MessageBottomSheetDialog in scala.

This removes some usage of old java api (mainly Message) and makes it more
future proof by handling concurrent ZMessaging instances properly.

To test it:
- go to messages list
- long press on different types of messages
- ensure that bottom menu is shown and proper actions are shown there, check that actions work correctly (also check ephemeral messages)
#### APK
[Download build #9151](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9151/artifact/build/artifact/wire-dev-PR978-9151.apk)
